osu.Game/Scoring/ScoreManager.cs

```diff
@@ -88,7 +92,7 @@ public IEnumerable<ScoreInfo> OrderByTotalScore(IEnumerable<ScoreInfo> scores)
         /// </remarks>
         /// <param name="score">The <see cref="ScoreInfo"/> to retrieve the bindable for.</param>
         /// <returns>The bindable containing the total score.</returns>
-        public Bindable<long> GetBindableTotalScore([NotNull] ScoreInfo score) => new TotalScoreBindable(score, configManager);
+        public Bindable<long> GetBindableTotalScore([NotNull] ScoreInfo score) => new TotalScoreBindable(score, configManager, multiplayerClient.APIRoom);
```

Comment:
Same goes for this. `GetBindableTotalScore()` should not need to know anything about any rooms, period.